src/aks-preview/setup.py

```diff
@@ -9,7 +9,7 @@
 
 from setuptools import setup, find_packages
 
-VERSION = "1.0.0b1"
+VERSION = "1.0.0b2"
```

Comment:
@FumingZhang also no version bump here?

Reply:
I suppose it's ok to bump the version here to trigger a new release